tests/test_core.py

```diff
@@ -4745,6 +4753,8 @@ def test_stat(self):
     src = open(path_from_root('tests', 'stat', 'test_stat.c')).read()
     self.do_run(src, 'success', force_c=True)
 
+    self.verify_in_strict_mode('src.c.o.js')
```

Comment:
Maybe an new test `test_strict_mode`  rather than conflating this wit stat?   I guess its because the failure was related to i64 on the boundary which stat uses?  Even so perhaps a separate test with a comment that says why stat and strict mode are tested together?

Reply:
I think we'll want multiple such strict mode tests - that's why I created the `verify_in_strict_mode` helper, which is called from the stat test. It should be easy to add to more places too. Basically, this is like closure compiler, we should run it from relevant tests here and there, I think.

Reply:
Actually, why don't we just run all our tests in struct mode?   I.e. add a --pre-js with strict to all tests?

Reply:
Might be worth looking into, but it could be a nontrivial amount of work, and as strict semantics are not identical to normal ones, it may be a different environment than our users run the code in which would be risky. But adding strict mode tests could be useful.